prow/plugins/plugins.go

```diff
@@ -64,6 +64,25 @@ var (
 	CommentMap, _              = genyaml.NewCommentMap()
 )
 
+// Status provides additional info about handler execution
+type Status struct {
+	// tookAction indicates whether the handler did any valuable work (e.g. API call)
+	tookAction bool
+}
+
+// TookAction sets the private s.tookAction value to true
+func (s *Status) TookAction() {
+	s.tookAction = true
+}
+
+// GetAction returns a string indicating whether the action was taken
+func (s *Status) GetAction() string {
+	if s.tookAction {
+		return "true"
+	}
+	return "false"
+}
```

Comment:
I'm not a Go guru myself, so take with a grain of salt: setters/getters are not that common in Go, but when they are present, the convention is that `TookAction` would be the getter and the setter would have a `Set`/`Record`/whatever name (https://go.dev/doc/effective_go#Getters)

Reply:
I know, but I wanted `tookAction` to be private, to have the state of status always false, and if the first occurrence of `action` then change to true and do not modify further. In the end, if action is taken why allow the user to modify it back? It's a more objective approach, please further comment if not needed here.
Regarding the naming, I can give whatever the names are needed but this particular getter is only for `Prometheus`, so maybe I can indicate that in a name?